recommendation_engine/config/params.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file defines a parameter class that contains the model hyperparameters.
+# Copyright © 2018 Avishkar Gupta
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+class Params():
+    """This class contains the model hyperparamters.
+    """
+    def __init__(self):
+        self.a = 1
```

Comment:
I think you can also define these variable as static variables this way you don't need to initialize params object